neuro_san/client/simple_one_shot.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Most cases will have this extra newline at the top to match what the automated tool expects